packages/shared/components/popups/EnableLedgerBlindSigningPopup.svelte

```diff
@@ -0,0 +1,51 @@
+<script lang="typescript">
+    import { onDestroy } from 'svelte'
+    import { Text, Icon } from 'shared/components'
+    import { localize } from '@core/i18n'
+    import { ledgerDeviceStatus } from '@core/ledger'
+    import { closePopup, openPopup } from '@lib/popup'
+
+    export let sendConfirmationPopupProps = null
+    export let mintNativeTokenPopupProps = null
```

Comment:
> I always feel like initializing something as null doesn't really make sense, since it could just as well be undefined. Maybe we should discuss this and create a coding convention out of it. Because sometime we initialize it as undefined or as null.

In my mind, I thought of `null` as a way of explicitly saying we're initializing this as an empty variable. It could be `undefined` but since that value / type is returned from most functions rather than `null`, using both keeps the intention and code a bit clearer IMO.

Reply:
After thinking about it, `undefined` does make more sense idiomatic to the language since declared but uninitialized values are `undefined`. But rather than initializing the variable as `undefined` explicitly, we just declare the object with a type and it will automatically be `undefined`. It's also slightly less code too.